lib/materialist/materializer/internals/link_href_mapping.rb

```diff
@@ -2,12 +2,17 @@ module Materialist
   module Materializer
     module Internals
       class LinkHrefMapping
-        def initialize(key:, as:)
+        def initialize(key:, as:, url_parser: nil)
           @key = key
           @as = as
+          @url_parser = url_parser
         end
 
         attr_reader :key, :as
+
+        def url_parser
+          @url_parser || ->url { url }
```

Comment:
it'd be nice to extract this block to something like `default_block` variable to make it more explicit.